framework/include/problems/FEProblemBase.h

```diff
@@ -352,6 +352,12 @@ class FEProblemBase : public SubProblem, public Restartable
    */
   unsigned int getMaxQps() const;
 
+  /**
+   * @return The number of nonlinear systems
+   */
+
+  unsigned int getNonLinearSize() const { return _nl.size(); }
```

Comment:
we have this api already?
```
  virtual std::size_t numNonlinearSystems() const override { return _num_nl_sys; }
```

Reply:
I removed it from FEProblemBase.h and used the other function instead to get the number of nonlinear systems.

Reply:
seems to be back now?

Reply:
I think I made a mistake when pushing my changes because my branch didn't seem to have that function, so I recommitted and tried pushing again. Hopefully that fixes the issue.